tests/php/test-class-amp-base-sanitizer.php

```diff
@@ -270,6 +271,46 @@ public function test_remove_invalid_child() {
 		AMP_Validation_Manager::$validation_results = null;
 	}
 
+	/**
+	 * Tests remove_invalid_child.
+	 *
+	 * @covers AMP_Base_Sanitizer::remove_invalid_child()
+	 * @covers AMP_Base_Sanitizer::has_dev_mode_exemption()
+	 */
+	public function test_remove_invalid_child_dev_mode() {
+		$html = '<div></div>';
+		/**
+		 * Element.
```

Comment:
This type of visual clutter is a result of the `Generic.Commenting.DocComment.MissingShort`, which is just broken in my opinion.
Might be one of the rules to consider for exclusion...